wezterm-gui/build.rs

```diff
@@ -11,7 +11,9 @@ fn main() {
             .ok()
             .and_then(|cwd| cwd.parent().map(|p| p.to_path_buf()))
             .unwrap();
-        let exe_output_dir = repo_dir.join("target").join(profile);
+        let exe_output_dir = std::env::var("CARGO_TARGET_DIR")
+            .and_then(|s| Ok(std::path::PathBuf::from(s)))
+            .unwrap_or(repo_dir.join("target").join(profile));
```

Comment:
Seems broken for the common case?  Should append `profile` to the effective target dir, whether it is set via env or not?

```suggestion
            .unwrap_or(repo_dir.join("target")).join(profile);
```